ipn/serve.go

```diff
@@ -62,6 +78,41 @@ type FunnelConn struct {
 	Src netip.AddrPort
 }
 
+// ServeStreamRequest defines the json request body
```

Comment:
JSON

Reply:
Addressed in https://github.com/tailscale/tailscale/pull/9014